dockerize.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# convenience script for creating/deleting docker files after compiling services
+
+make_images () {
+services=("eureka-server" "config" "Spring-Cloud-Gateway" "accounts" "inventory" "commerce")
+
+for ((i=0; i<${#services[@]}; i++)); do
+	cd ${services[i]}
+	#mvn clean package
+	if [ ${services[i]} = "Spring-Cloud-Gateway" ]; then
+		echo 1
+		docker build -t 'aypas/spring-cloud-gateway' .
+	else
+		echo 2
+		docker build -t aypas/${services[$i]} .
+	fi
+	echo `ls`
+	cd ..
+done
+}
```

Comment:
1. the mvn line is commented out, so won't this break if that's not manually run in advance?
2. the mvn part should really just be inside the dockerfile, no? and the rest of this, i do like the initiative, but isn't it just trying to do the same job docker-compose can already do?

Reply:
response expected to prior feedback. also, why are those echos here?